.github/workflows/ci.yml

```
@@ -1369,6 +1364,85 @@ jobs:
+        run: pnpm start engines download typedb 3
+
+      - name: Show installed engines
+        run: pnpm start engines list
+
+      - name: Run TypeDB integration tests
+        run: pnpm test:engine typedb
+        timeout-minutes: 15
+
+      - name: Show TypeDB logs on failure (Unix)
+        if: failure() && runner.os != 'Windows'
+        run: |
```

actionlint (1.7.10)

[error] shellcheck reported issue in this script: SC2010:warning:5:1: Don't use ls | grep. Use a glob or a for loop with a condition to allow non-alphanumeric filenames

(shellcheck)